projects/addon-doc/index.ts

```diff
@@ -29,7 +28,6 @@ export const TuiAddonDoc = [
     TuiDocPageTabConnector,
     TuiDocNavigation,
     TuiDocMain,
-    TuiDocText,
```

Comment:
Maybe we should deprecate and keep it? In case somebody was using it

Reply:
fixed